exo-setup/features/steps/steps-then.ls

```diff
@@ -28,3 +29,20 @@ module.exports = ->
 
   @Then /^it finishes with exit code (\d+)$/ (+expected-exit-code) ->
     expect(@process.exit-code).to.equal expected-exit-code
+
+
+  @Then /^ExoCom uses this routing:$/ (table) ->
+    expected-routes = []
+    for row in table.hashes!
+      service-routes = {role: row.ROLE}
+      for message in row.RECEIVES.split(', ')
+        (service-routes.receives or= []).push message
+      for message in row.SENDS.split(', ')
+        (service-routes.sends or= []).push message
+      if row.NAMESPACE
+        service-routes.namespace = row.NAMESPACE
+      expected-routes.push service-routes
+    docker-config = yaml.safe-load fs.read-file-sync(path.join(@current-dir, 'tmp', 'docker-compose.yml'))
+    actual-routes = JSON.parse docker-config.services['exocom0.21.8'].environment.SERVICE_ROUTES
+    jsdiff-console actual-routes, expected-routes
+
```

Comment:
this empty line should be removed. I have an editor setting that makes sure a file ends with only a single trailing newlines